compiler/luci/import/src/GraphBuilderRegistry.cpp

```diff
@@ -161,6 +161,13 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   // BuiltinOperator_ARG_MAX = 56,
   // BuiltinOperator_HARD_SWISH = 117,
   // BuiltinOperator_DENSIFY = 124,
+
+#define CIRCLE_VNODE(OPCODE, CLASS) add(std::make_unique<CLASS>())
+
+  // TODO add CIRCLEINPUT and CIRCLEOUTPUT builder
+  CIRCLE_VNODE(CIRCLECONST, CircleConstNodeBuilderBase);
```

Comment:
Why is it `CircleConstNodeBuilderBase`?

Reply:
I thought name `Base` is to for base class and you used for implementation.
Plese don't confuse using names.

Reply:
Ahh, better to change to `CircleConstNodeBuilder`. `Base` added accidentally, sorry.

Reply:
Fixed!